addons/binding/org.openhab.binding.network.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name:  network.test Binding Tests
+Bundle-SymbolicName: org.openhab.binding.network.test;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Fragment-Host: org.openhab.binding.network
+Import-Package: org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.util,
+ org.hamcrest;core=split,
+ org.junit;version="4.0.0",
+ org.mockito,
+ org.openhab.binding.network,
+ org.openhab.binding.network.handler,
+ org.osgi.framework,
+ org.osgi.service.device,
+ org.slf4j
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
I do not see it in the source file of skeleton:

https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

I think you should fix the other places where you used it as well.